packages/chains/src/arbitrum/assets.ts

```diff
@@ -122,6 +125,32 @@ export const assets: SupportedAsset[] = [
     oracle: OracleTypes.ChainlinkPriceOracleV2,
     extraDocs: defaultDocs("https://arbiscan.io", USDT),
   },
+  {
+    symbol: assetSymbols.USDs,
+    underlying: USDs,
+    name: "Sperax USD",
+    decimals: 18,
+    oracle: OracleTypes.UniswapV3PriceOracle,
+    extraDocs: defaultDocs("https://arbiscan.io", USDs),
+    disabled: true,
+  },
+  // Awaiting price feeds
```

Comment:
what price feeds are awaiting here if they are uni v3 and can use the embedded price feed?